keyboards/sendyyeah/pix/local_drivers/i2c.c

```diff
@@ -0,0 +1,159 @@
+#include <util/twi.h>
```

Comment:
Would prefer that all of these files were removed.  

They're for split keyboard support, mostly, and deprecated in favor of the split keyboard feature. 

Additionally, the i2c and serial drivers exist in core and should be used (drivers/(arch)/i2c_master.c, for instance).  And those can be used here, instead.

Reply:
done